docs/application/web/api/6.5/device_api/mobile/tizen/ml.html

```diff
@@ -595,7 +595,7 @@ <h4>Methods</h4>
 <p><span class="except">Exceptions:</span></p>
           <ul class="exception"><li>WebAPIException<ul>
 <li class="list"><p>
- with error type InvalidValuesError, if any of the input parameters contain an invalid value.
+ with error type InvalidValuesError, if any of the input parameters contains an invalid value.
```

Comment:
Global: I missed suggesting removing S for parameters, can we remove S from contains, revert back to original is better in this context.
with error type InvalidValuesError, if any of the input parameters contain an invalid value.